build/package/Dockerfile.gradle-toolset

```diff
@@ -25,15 +25,14 @@ RUN cd /opt && \
     echo "Loading gradle cache with gradlew  ${GRADLE_VERSION} distribution" && \
     mkdir -p /tmp/temp-gradle-app && cd /tmp/temp-gradle-app && touch settings.gradle && \
     gradle wrapper --gradle-distribution-sha256-sum ${GRADLE_WRAPPER_DOWNLOAD_SHA256} && ./gradlew -version && \
-    mkdir -p /home/gradle && \
-    chown -R 1001:0 /home/gradle $HOME && \
-    chmod -R g=u /home/gradle $HOME && \
-    ln -s /root/.gradle ${GRADLE_USER_HOME}
+    chown -R 1001:0 /workspace/source $HOME && \
+    chmod -R g=u /workspace/source $HOME
+
+VOLUME /workspace/source
 
 # Add proxy settings
 COPY build/package/scripts/set_gradle_proxy.sh /usr/local/bin/set_gradle_proxy.sh
-RUN chmod +x /usr/local/bin/set_gradle_proxy.sh && \
-    /usr/local/bin/set_gradle_proxy.sh
+RUN chmod +x /usr/local/bin/set_gradle_proxy.sh
```

Comment:
Could we merge the two lines above into the next section which already copies scripts and makes them executable?

Reply:
While we're at it, can we rename the script to use dashes instead of underscores? All other scripts use underscores.